### ISSUE TITLE
handing screenshot data

### ISSUE THREAD

**Issue by zoi-aoba** [blendlabs/react-native-screenshot-detector#4]
Can I handle the screenshot data?

**Comment by gcarling** [comment]
No, at the moment this is set up just to let you know that a screenshot has been taken. It may be possible to do this - if you think you have found a way, feel free to open up a pull request and I'll look it over.

**Comment by zoi-aoba** [comment]
Thanks for your response.
I will try to find the way :) And if I could, submit PR!

**Comment by EricPKerr** [comment]
@goleador0322 you could combine this with https://github.com/gre/react-native-view-shot to programmatically capture the screenshot data yourself when the `ScreenshotDetector.subscribe` callback event fires

**Comment by zoi-aoba** [comment]
@EricPKerr thanks :) I will try.
